### PR TITLE
Valentina mignosa

### PR DESCRIPTION
**Pull request by valentinamignosa**

### CODE REVIEW

File: inscriptions/ISic1087.xml [hunk 1]
```diff
@@ -181,154 +186,24 @@
...
-                </supplied> ἔλα<supplied reason="lost">φο</supplied>ν, τὰ <supplied reason="lost">δ’ ἔτ’</supplied> ὀνύματα
-                    ταῦτα κολ-<lb n="9"/>άψαντ<supplied reason="lost">ες, ἐς</supplied> τὸ Ἀ<supplied reason="lost">
-                    π</supplied>ολλόˉνιον καθθέμε-<lb n="10"/>ν, τὸ Διὸ<supplied reason="lost">ς
-                    προ</supplied>γρά<supplied reason="lost">ψ</supplied>αντες. τὸ δὲ χρυσίον<lb n="11"/>ἑξέˉκ<supplied reason="lost">οντα τα</supplied>λάντοˉν <supplied reason="lost">ἐ͂</supplied>μεν.
-                    <lb n="12"/>
+                    <lb n="1"/><supplied reason="lost">δι</supplied>ὰ το&#7621;ς θεο&#7621;ς τṓ<supplied reason="lost">σ</supplied>δε νικο&#x0342;ντι τοὶ Σελινο&#7620;ν<supplied reason="lost">τιοι</supplied>·
+                    <lb n="2"/><supplied reason="lost">δι</supplied>ὰ τὸν Δία νικ&#x0342;μες καὶ διὰ τὸν Φόβον <supplied reason="lost">καὶ</supplied>
+                    <lb n="3"/><supplied reason="lost">δ</supplied>ιὰ hερακλέα καὶ δι’ Ἀπόλλο&#772;να καὶ διὰ Π<supplied reason="lost">οτ</supplied>
```
Comment (simonastoyanova) [inline inscriptions/ISic1087.xml:193]: Is that a Latin 'h' in Herakles? Will think what to do with it.

File: inscriptions/ISic1087.xml [hunk 2]
```diff
@@ -181,154 +186,24 @@
...
+                    <lb n="4" break="no"/>ει<supplied reason="lost">δ&#x0342;</supplied>να καὶ διὰ Τυνδαρίδας καὶ δι᾿ Ἀθ<supplied reason="lost">α</supplied>
+                    <lb n="5" break="no"/>ναίαν καὶ διὰ Μ<supplied reason="lost">α</supplied>λοφόρον καὶ διὰ Πασι<supplied reason="lost">κ</supplied>
+                    <lb n="6" break="no"/>ρά<supplied reason="lost">τ</supplied>ειαν καὶ διὰ το&#7621;ς ἄλλο&#772;ς θεο&#7621;ς, διὰ δ<supplied reason="lost">ὲ</supplied> Δία
+                    <lb n="7"/>μάλιστα. φιλίας δὲ γενομένας ἐν χ<supplied reason="lost">ρ</supplied>ύσ
+                    <lb n="8" break="no"/>εο&#772;ι ἐλά<supplied reason="lost">με</supplied>ν τὰ <supplied reason="lost">δ᾿ ἔτι</supplied> ὀνύματα ταῦτα κόλ
+                    <lb n="9" break="no"/>άψαντ<supplied reason="lost">ες εἰς τ</supplied>ὸ <supplied reason="lost"> Ἀπο</supplied>λλο&#7620;νιον κάθθεμε
```
Comment (simonastoyanova) [inline inscriptions/ISic1087.xml:199]: Rogue space in the supplied in Apollo.

File: inscriptions/ISic1057.xml [hunk 3]
```diff
@@ -178,39 +183,17 @@
...
-                    </expan>
-                    ιʹ
-                    <expan>
-                        <abbr>κ</abbr>
-                        <ex>α</ex>
-                        <abbr>λ</abbr>
-                    </expan>
-                    (άνδαις) Φεβλαρίες <surplus>²⁶Φεβρουαρίαις</surplus>²⁶.<lb n="5"/> μή τις ἀνύξῃ.
-                    <lb n="6"/>
+                    <lb n="1"/>ἐντάδε κῖτε πε&#x0342;ς
```
Comment (JonPrag) [inline inscriptions/ISic1057.xml:188]: ἐνθάδε

File: inscriptions/ISic1057.xml [hunk 4]
```diff
@@ -178,39 +183,17 @@
...
+                    <lb n="2"/>ὀνόματι Κλωδια
+                    <lb n="3" break="no"/>νός · τελευτᾷ <expan><abbr>μη
+                    <lb n="4" break="no"/>ν</abbr><ex>ῶν</ex></expan> <hi rend="supraline"><num value="10">ι</num></hi> <expan><abbr>κ</abbr><ex>α</ex><abbr>λ</abbr><ex>άνδαις</ex></expan> Φεβλαρίες
```
Comment (JonPrag) [inline inscriptions/ISic1057.xml:191]: There is no supraline above the numeral on this text. I note the abbreviation / ligature for Kalandais is actually much more complex, but that can be left for future editing!

File: inscriptions/ISic1074.xml [hunk 5]
```diff
@@ -178,42 +183,18 @@
...
-                <ab>
-                    <lb n="1"/>
-
-
-                     . . . ἐποί<supplied reason="omitted">η</supplied>-<lb n="2"/>σε
-                    δὲ θυγάτηρ ἰδίᾳ μητρὶ εὐσεβῇ.
-                    <lb n="3"/>
+                    <lb n="1"/>Λανθάνουσα <g type="interpunct">·</g> <supplied reason="omitted">ἡ</supplied>καὶ Ἀγάθη χρη
+                    <lb n="2" break="no"/>στὴ καὶ ἄμεμπτος <g type="interpunct">·</g> ἔζησεν <g ref="http://sicily.classics.ox.ac.uk/alists/listglyph.xml#leaf">❦</g>
+                    <lb n="3"/>ἔτη <hi rend="supraline"><num value="72">οβ</num></hi> μῆνες <hi rend="supraline"><num value="4">δ</num></hi> ἔτι καὶ <hi rend="supraline"><num value="3">γ</num></hi> · ἐποί<supplied reason="omitted">η</supplied>
```
Comment (JonPrag) [inline inscriptions/ISic1074.xml:190]: supraline is only over the omicron, not the beta as well, in th first numeral; none over the second numeral (delta), or the third (gamma). NB the supraline is recording of a feature on the text, not simply part of the default mark-up of a numeral. The ' included in edited texts is simply indicator of a numeral (equivalent to our `<num>` element), not a reflection of the existence of a supraline or similar.

**Comment by valentinamignosa** [comment]
I put the supraline systematically... I was thinking (don't know why) it was a feature of the mark up of greek number... I will check the other files

File: inscriptions/ISic1087.xml [hunk 6]
```diff
@@ -181,154 +186,23 @@
...
-                    <lb n="12"/>
+                    <lb n="1"/><supplied reason="lost">δι</supplied>ὰ το&#7621;ς θεο&#7621;ς τṓ<supplied reason="lost">σ</supplied>δε νικο&#x0342;ντι τοὶ Σελινο&#7620;ν<supplied reason="lost">τιοι</supplied>·
+                    <lb n="2"/><supplied reason="lost">δι</supplied>ὰ τὸν Δία νικ&#x0342;μες καὶ διὰ τὸν Φόβον <supplied reason="lost">καὶ</supplied>
+                    <lb n="3"/><supplied reason="lost">δ</supplied>ιὰ hερακλέα καὶ δι’ Ἀπόλλο&#772;να καὶ διὰ Π<supplied reason="lost">οτ</supplied>
+                    <lb n="4" break="no"/>ει<supplied reason="lost">δ&#x0342;</supplied>να καὶ διὰ Τυνδαρίδας καὶ δι᾿ Ἀθ<supplied reason="lost">α</supplied>
+                    <lb n="5" break="no"/>ναίαν καὶ διὰ Μ<supplied reason="lost">α</supplied>λοφόρον καὶ διὰ Πασι<supplied reason="lost">κ</supplied>
+                    <lb n="6" break="no"/>ρά<supplied reason="lost">τ</supplied>ειαν καὶ διὰ το&#7621;ς ἄλλο&#772;ς θεο&#7621;ς, διὰ δ<supplied reason="lost">ὲ</supplied> Δία
+                    <lb n="7"/>μάλιστα. φιλίας δὲ γενομένας ἐν χ<supplied reason="lost">ρ</supplied>ύσ
```
Comment (JonPrag) [inline inscriptions/ISic1087.xml:197]: φιλίας should be φιλία[ς], i.e. `<supplied reason="lost">ς</supplied>`

**Comment by JonPrag** [comment]
Okay, gone through, a few minor things (the most substantive is obviously the supraline point). When those things tweaked, commit is good to go.

**Comment by valentinamignosa** [comment]
Ok, thanks

**Review by JonPrag — APPROVED** [review]
all good!

**Pull request merged**